lms/djangoapps/support/urls.py

```diff
@@ -45,8 +49,12 @@
         FeatureBasedEnrollmentSupportAPIView.as_view(),
         name="feature_based_enrollment_details"
     ),
-    re_path(r'link_program_enrollments/?$', LinkProgramEnrollmentSupportView.as_view(),
-            name='link_program_enrollments'),
+    re_path(r'link_program_enrollments/?$', LinkProgramEnrollmentSupportView.as_view(), name='link_program_enrollments'),
```

Comment:
nit: this line is causing quality failure.

Reply:
Updated.